src/InterestPayout.Worker/Startup.cs

```diff
@@ -58,6 +69,7 @@ protected override void ConfigureServicesExt(IServiceCollection services)
                     });
                 })
                 .AddMessaging(Config.RabbitMq)
+                .AddHostedService<CqrsMessagingInitializer>()
```

Comment:
Since MT recurring message consumer depends on Lykke's CQRS engine. It makes sense to initialize CQRS engine first

Reply:
`+`